.goreleaser.yaml

```diff
@@ -38,10 +38,14 @@ builds:
     tags:
       - ledger
       - multiplexer
+    flags:
+      - -trimpath
     ldflags:
       # Ref: https://goreleaser.com/customization/templates/#common-fields
       # .Version is the version being released
       # .FullCommit is git commit hash goreleaser is using for the release
+      - -s
+      - -w
```

Comment:
These are linker flags passed to go build:
	•	-s: Omit the symbol table (used for debugging, stack traces, profiling).
	•	-w: Omit DWARF debugging info (used by debuggers like dlv).

By doing both, we decrease the size of the built binary.